packages/subgraph/tests/osx/pluginSetupProcessor.test.ts

```diff
@@ -1,5 +1,3 @@
-import {generatePluginInstallationEntityId} from '../../commons/ids';
-import {createInstallationPreparedEvent} from '../../commons/test';
 import {PLUGIN_REPO_ADDRESS} from '../../imported/repo-address';
 import {handleInstallationPrepared} from '../../src/osx/pluginSetupProcessor';
```

Comment:
As a task for the future, maybe we want to add some root aliases to the repo so we don't have to climb back up the file structure (e.g. using ~commons/ids)